apps/mobile/src/components/ContextMenu/ContextMenu.js

```diff
@@ -18,14 +18,15 @@ export default function ContextMenu () {
   const insets = useSafeAreaInsets()
   const navigation = useNavigation()
   const { t } = useTranslation()
+  const openURL = useOpenURL()
```

Comment:
is this how we should be using openURL from here on out?

Reply:
Yes, always preferable, as the non-hook version relies upon a global navigationRef vs getting it through context. The difference is academic for now, but the global ref could become unstable. So yes, but this is just clean-up though, I wouldn't fret much about it.